components/application/index.jsx

```diff
@@ -11,9 +10,7 @@ const Application = ({ children, dataProvider, activity, ...restProps }) => (
   <>
     <Head />
     <Container {...restProps}>
-      <AppBar>
-        <RepositorySelect value={dataProvider} />
-      </AppBar>
+      <AppBar dataProvider={dataProvider} />
```

Comment:
Why could not you keep this as it was?

Reply:
I needed component connected to `LayoutContext`. (I didn't want to connect another component into it.)